test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_config_test.clj

```diff
@@ -93,8 +103,37 @@
 
     (is (expected-https-config?
           (merge valid-ssl-pem-config
-                 {:ssl-host "foo.local" :ssl-port 8001 :request-header-max-size 16192})
-          {:https {:host "foo.local" :port 8001 :request-header-max-size 16192}})))
+                 {:ssl-host "foo.local"
+                  :ssl-port 8001
+                  :request-header-max-size 16192})
+          {:https {:host "foo.local"
+                   :port 8001
+                   :request-header-max-size 16192}}))
+
+    (is (expected-https-config?
+          (merge valid-ssl-pem-config
+                 {:ssl-host "foo.local"
+                  :ssl-port 8001
+                  :so-linger-seconds 22})
```

Comment:
I'm not sure it's _necessary_, but I have definitely been in situation in the past where I've been frustrated that a piece of software wouldn't let me configure the value I'd calculated as "optimum."

For example, with a load balancer handling thousands of connections we may know that 99.9% of RTT's complete within 150ms, so a 200ms timeout would be more than sufficient to cover "normal" traffic patterns.  However, 1000ms timeout may mean that hundreds (possibly thousands) of connections linger around longer than they need to because the system is handling thousands of connections _per second_, and we're keeping them around 800ms longer than we need to.

Again, I don't have a concrete use case right now, it's just something I've run into working on things at scale previously.  With NGINX, for example, we were easily able to hit 2,500 requests _per second_ with an x1.large in EC2, and many more thousands of open connections at any given time.